src/PerfView/PerfViewData.cs

```diff
@@ -4122,36 +4123,54 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
             }
             else if (streamName == "CCW Ref Count")
             {
-                // TODO use the callback model.  We seem to have an issue getting the names however. 
-                foreach (var data in events.ByEventType<CCWRefCountChangeTraceData>())
+                var interopTraceEventParser = new InteropTraceEventParser(eventSource);
+                interopTraceEventParser.AddCallbackForEvents<TaskCCWQueryRuntimeClassNameArgs>(args =>
```

Comment:
It turns out that this QueryClassName event is probably not going to be useful to get a symbolic name, and it certainly should not be representing a 'increment' (which is what Metric=1) will do.   Just remove it.

Reply:
Thanks, done.